docs/cryptodoc/src/05_09_frodokem.rst

```diff
@@ -0,0 +1,272 @@
+   +----------------------------------------------------------------+-------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`Modes <pubkey/frodokem/modes>`                           | :srcref:`src/lib/pubkey/frodokem/frodo_mode.h`        | Parameter set representation                                                                                                                                                           |
+   +----------------------------------------------------------------+-------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`Constants <pubkey/frodokem/modes>`                       | :srcref:`src/lib/pubkey/frodokem/frodo_constants.h`   | Parameter set instantiations                                                                                                                                                           |
+   +----------------------------------------------------------------+-------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`Matrix Operations <pubkey/frodokem/matrix_operations>`   | :srcref:`src/lib/pubkey/frodokem/frodo_matrix.h`      | Matrices and operations on them                                                                                                                                                        |
+   +----------------------------------------------------------------+-------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | FrodoKEM                                                       | :srcref:`src/lib/pubkey/frodokem/frodokem.h`          | FrodoKEM :ref:`Key Generation <pubkey/frodokem/key_generation>`, :ref:`Encapsulation <pubkey/frodokem/encapsulation>`, :ref:`Decapsulation <pubkey/frodokem/decapsulation>`            |
+   +----------------------------------------------------------------+-------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Frankly, I think we shouldn't go into such detail. The code is approachable enough that people will find their way around it, if they have the need to do so. There's not much this table adds to the documentation and we'll need to remember to keep this table updated if things change. 🤡